legacy/Dockerfile

```diff
@@ -1,18 +0,0 @@
-# Dockerfile for Alcor Controller
-
```

Comment:
Why would you remove the legacy directory?

Reply:
my bad, I put it back in 91ffb7d